tests/test_googlehits.py

```diff
@@ -151,10 +151,13 @@ def test_fetch(self, mock_utcnow):
 
         self.assertEqual(len(items), 1)
         item = items[0]
+
+        import json
```

Comment:
Don't import this in the middle of the code, please. Moreover, don't print anything when tests are run.